ZTP.md

```diff
@@ -52,6 +59,35 @@ Use case: large scale deployments (where automation and security are major drive
 - Need to consider mDNS as another option if DHCP is absent.
 - Need to consider LLDP/SSDP as well.
 
+#### Re-Provisioning
+
+Should there be a ztp setting that automatically goes back into ZTP mode if the network changes? So if you do an initial ZTP and are on 192.168.1.0/24 but then you are unplugged and plugged back in and your DHCP shows you 10.0.0.0/24, ZTP automatically assumes the DPU has been either sold, moved datacenters, reprovisioned, etc and retriggers ZTP?
```

Comment:
Does this long-line pass the markdown linter?

Reply:
yes, since I disabled this rule everywhere on our linters...